recipes/_initial.rb

```diff
@@ -25,5 +25,4 @@
   hostname 'localhost'
   comment 'Appended by st2 cookbook'
   action :append
```

Comment:
Not sure about that entry if we need it at all. At least after using `127.0.0.1` instead of `localhost` this rule is out of scope of this cookbook.

From one point of view cleanup might be good, from another side: `127.0.0.1 localhost` in hosts won't hurt.

@shortdudey123 Ideas?